app/Locale/German.php

```diff
@@ -311,8 +305,8 @@ public function get_locale(): array
 			'SETTINGS_SUCCESS_MAP_DISPLAY' => 'Karteneinstellungen erfolgreich aktualisiert',
 			'SETTINGS_SUCCESS_MAP_DISPLAY_PUBLIC' => 'Karteneinstellungen für öffentlichen Alben erfolgreich aktualisiert',
 			'SETTINGS_SUCCESS_MAP_PROVIDER' => 'Provider für Karten erfolgreich aktualisiert',
-			'SETTINGS_SUCCESS_CSS' => 'CSS aktualisiert',
-			'SETTINGS_SUCCESS_UPDATE' => 'Einstellungen erfolgreich aktualisiert',
+			'SETTINGS_SUCCESS_CSS' => 'CSS updated',
+			'SETTINGS_SUCCESS_UPDATE' => 'Settings updated successfully',
```

Comment:
Why replacing a totally fine German translation by the English original?

Reply:
Mistake. Fixed [5d3a3ef](https://github.com/LycheeOrg/Lychee/pull/1539/commits/5d3a3eff43046bc0fb28645b66d6055346132b66).